package.json

```diff
@@ -26,7 +26,6 @@
     "@types/jquery": "^3.3.30",
     "@types/node": "^13.1.2",
     "@types/offscreencanvas": "^2019.6.1",
-    "@webgpu/glslang": "^0.0.15",
```

Comment:
(huh I couldn't comment on the lower line)
nit: I don't think we need `babel-plugin-macros` anymore. Same comment in #150 as well